modules/core/src/main/java/org/eclipse/imagen/GraphicsJAI.java

```diff
@@ -428,6 +428,18 @@ public void dispose() {
         g.dispose();
     }
 
+    /**
+     * Graphics has a finalize method that automatically calls dispose()
+     * for subclasses.  We override it here with an empty method so that
+     * it doesn't dispose the private Graphics2D field.
+     *
+     * @see <a href="https://web.archive.org/web/20110412105630/
+     * http://java.net/jira/browse/JAI_CORE-96">JIRA JAI-CORE 96</a>
```

Comment:
To make the URL work I needed to remove the space:


https://web.archive.org/web/20110412105630/http://java.net/jira/browse/JAI_CORE-96